presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentModule.java

```diff
@@ -21,6 +21,7 @@
 import io.prestosql.tests.product.launcher.env.common.Standard;
 import io.prestosql.tests.product.launcher.env.environment.Multinode;
 import io.prestosql.tests.product.launcher.env.environment.Singlenode;
+import io.prestosql.tests.product.launcher.env.environment.SinglenodeCassandra;
```

Comment:
> Move Cassandra Product Tests to launcher

This is the commit pattern i used. Not very important, but since this is a part of a series, we may choose to be consistent:

```
Convert cassandra environment to Launcher
```

(my commits didn't have "environment", but it seems we better add it, especially in this case)